docs/docs/networking.md

```diff
@@ -298,6 +298,47 @@ Specify `container/bridge` mode through the `networks` property:
     }
   },
 ```
+##### Installing the `mesos-bridge` CNI plugin
+If you are not using DC/OS and want to enable `container/bridge` mode with the Universal Container Runtime (UCR), several more steps are necessary to install and use the `mesos-bridge` CNI plugin.
+
+**Prerequisites**
+- Mesos version 1.2.0 or higher.
+- Marathon version 1.5 or higher.
+
+1. Navigate to or create a `/var/lib/mesos/cni` directory on each of your agent nodes as well as `config` and `plugins` subdirectories.
+
+1. Clone the CNI repository from [https://github.com/containernetworking/cni](https://github.com/containernetworking/cni) and build according to their instructions.
```

Comment:
I'd make this clone-and-build the first step:

1. It's likely the step that will take the longest.
2. The remaining steps become "create this file/dir" and "copy these things there", logically grouped together
3. Currently, it's not super clear if I'm supposed to clone the repo into the directory structure I just created.

Reply:
can do.